docs/01-kommunikationsmodelle/02-learning-goals.adoc

```diff
@@ -44,9 +44,9 @@ Die Teilnehmer:innen verstehen, dass sie durch regelmäßige Reflektion eine Kla
 
 Die Teilnehmer:innen können typische Stakeholder aus Sicht von Softwarearchitekt:innen identifizieren (z.B. Entwickler:innen, Management, Betrieb, Kunden) und deren Anliegen verstehen (z.B. via Maslowsche Bedürfnishierarchie).
 
-Sie können eine Stakeholderanalyse durchführen (z.B.: Modelle und Vorgehen einer Stakeholderanalyse kennen, z.B.: Stakeholder-Map, Beziehungs-Map bzw. Abhängigkeitsanalyse, RACI-Matrix
+Sie können eine Stakeholderanalyse durchführen (z.B. Modelle und Vorgehen einer Stakeholderanalyse kennen, z.B. Stakeholder-Map, Beziehungs-Map bzw. Abhängigkeitsanalyse, RACI-Matrix
```

Comment:
Die schließende Klammer fehlt immer noch 😛

Reply:
Uch behebe die Fehler nacheinander. Die Klammer war noch nicht dran ;-)

Ist jetzt aber eh weg wegen Aufzählung statt Liste in Klammern.